### ISSUE TITLE
Unable to define the Accessibility Focus order in React Native components

### ISSUE THREAD

**Issue by oargaruna** [facebook/react-native#13152]
### Description

There is no way to define the accessibility focus order for individual components. From what I can see when swiping left/right with Talkback/VoiceOver enabled is that the UI is scanned from top-left to bottom-right and accessible views are highlighted as they are encountered. I would like to define my own focus order for the screen readers, but currently there seems to be no way to do so.

### Additional Information

* React Native version: 0.34
* Platform: both
* Development Operating System: MacOS
* Dev tools: XCode 8.2.1/Android Studio 2.3

**Comment by cannona** [comment]
Looks like doing this on IOS involves subclassing the parent view? http://stackoverflow.com/questions/13279498/change-order-of-read-items-with-voiceover

This page also has a bit of info for both IOS and Android. http://www.bbc.co.uk/guidelines/futuremedia/accessibility/mobile/focus/focus-order

**Comment by hramos** [comment]
Hi there! This issue is being closed because it has been inactive for a while. Maybe the issue has been fixed in a recent release, or perhaps it is not affecting a lot of people. Either way, we're automatically closing issues after a period of inactivity. Please do not take it personally!

If you think this issue should definitely remain open, please let us know. The following information is helpful when it comes to determining if the issue should be re-opened:

- Does the issue still reproduce on the latest release candidate? Post a comment with the version you tested.
- If so, is there any information missing from the bug report? Post a comment with all the information required by the [issue template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md).
- Is there a pull request that addresses this issue? Post a comment with the PR number so we can follow up.

If you would like to work on a patch to fix the issue, *contributions are very welcome*! Read through the [contribution guide](http://facebook.github.io/react-native/docs/contributing.html), and feel free to hop into [#react-native](https://discordapp.com/invite/0ZcbPKXt5bZjGY5n) if you need help planning your contribution.

**Comment by Lily418** [comment]
I think this is still an issue

iOS View's have an [accessibilityElements](https://stackoverflow.com/questions/13279498/change-order-of-read-items-with-voiceover) property which allows you to change the VoiceOver read order.

Android has [accessibilityTraversalAfter](https://stackoverflow.com/questions/26729909/modify-accessibility-focus-order) which is a different approach to being able to change the read order.

I can't see any reference to these in the [React Native Accessibility](https://facebook.github.io/react-native/docs/accessibility.html) documentation, it would be great if there was a way to use these in React Native.

**Comment by LightScry** [comment]
@hramos can we re-open this?

**Comment by sospedra** [comment]
We definitely need this
